www/src/examples/Overlays/OverlayTrigger.js

```diff
@@ -1,4 +1,11 @@
-const renderTooltip = props => (
+const renderTooltip = ({
```

Comment:
we can do simpler than this:

```js
function renderTooltip(props) {
  return <Tooltip {...props}>Simple tooltip</Tooltip>;
}
```

and change `placement` below to just `right` instead of `right-start`

Reply:
Thank you for the suggestion! Implemented it. Do you know if this still resolves the issue in #312?